lib/content-services/src/lib/content-node-share/content-node-share.dialog.scss

```diff
@@ -57,16 +114,34 @@
 
     .mat-dialog-actions {
         justify-content: flex-end;
+        padding: 0;
+        margin: 8px 0 0 0;
 
         & > button {
-            text-transform: uppercase;
+            color: var(--adf-theme-foreground-base-color);
+            background-color: var(--theme-grey-text-background-color);
+            margin-right: 12px;
         }
     }
 
     .mat-form-field-flex {
         align-items: center;
     }
 
+    .mat-dialog-container {
+        padding: 0px;
```

Comment:
no need to mention 'px'. If you have 0 value then it is equal for each unit like 0px = 0% = 0vh etc. so you can have just 0.